### ISSUE TITLE
Add warning about old version

### ISSUE THREAD

**Issue by Arfey** [aio-libs/create-aio-app#104]
If at run of create-air-app a new version is available, then output a warning. ⚠️

**Comment by ivernus** [comment]
Were you thinking of a quick poll to an endpoint to check the version at startup @Arfey?

**Comment by Arfey** [comment]
Yes. Of course, if user will not have a connection to the internet he also will can create a template without problems.

**Comment by ivernus** [comment]
> Yes. Of course, if user will not have a connection to the internet he also will can create a template without problems

Do you think the following logic would work? 

1. User starts application 
2. A function makes a simple HTTP poll to pypi for the package version, if connection fails, skip to step 4
3. Compare package version with existing version and output warning if applicable
4. Application starts

**Comment by Arfey** [comment]
If `Application` in your logic is `create-aio-app` so yes.

**Comment by ivernus** [comment]
I added an initial test check here @Arfey https://github.com/ivernus/create-aio-app/commit/4b7ec5d8bbb8a4940d1162fa44775426e9e75ce8. But for some reason when I try to run the make test on the code dir it complains about missing requests library. I have added it to the requirements, tried reinstalling manually but it doesn't appear to work, also tried adding to the setup.cfg file as a requirement.

**Comment by Arfey** [comment]
Hi, could u make a pull request with ur changes? 😀

**Comment by priyanshu20** [comment]
hi,
can i work on this if it is still relevant?

**Comment by Arfey** [comment]
@ivernus please make a pull request with changes

**Comment by Arfey** [comment]
@priyanshu20 i hope it's not relevant 😀
but if u wanna help, u can add handlers for 500 and 404 error

**Comment by ivernus** [comment]
Sorry @Arfey i wasn't getting these notifications for some reason. Let me make the updates and send the PR

**Comment by Atharva-Aloni** [comment]
hey can I work on this issue or it is closed or assigned already ?
